packages/react-scripts/template/src/App.css

```diff
@@ -14,6 +14,10 @@
   color: white;
 }
 
+.App-header h1 {
```

Comment:
We intentionally don't have any tag-based selectors. Generally this is not recommended in React apps because those are very brittle. Can you please add an explicit className for it if this is necessary?

Reply:
Thanks for the feedback Dan! I pushed a commit with these changes.